docs/platforms.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Tested Platforms
```

Comment:
Perhaps this title?
> Tested Kubernetes Platforms

Because, the other platforms are desktop/mobile environments eg. Ubuntu/MacOS/etc.

Reply:
I did wonder if we should add the other platforms in the list as well. It would be as extensive a list, since most platform diversity is on Linux, and I think we should ultimate promote Flatpak for that.
If we're not adding the desktop platforms, then I agree with changing the title.

Let's hear from @vbatts about whether this is the format he had envisioned.

Reply:
Maybe related to this topic is CPU architecture... There's x64, but no armX docker images for example.